lib/assignDisabledRanges.js

```diff
@@ -117,14 +117,20 @@ module.exports = function (root, result) {
 
 	return result;
 
+	/** @typedef {import('postcss').Comment & { raws: { inline?: boolean } }} PostcssSassComment */
```

Comment:
[note] Extracting the `PostcssSassComment` type is for readability of `isInlineComment()`.